ores/lock_manager/lock_manager.py

```diff
@@ -0,0 +1,15 @@
+"""Lock manager interface."""
+
+
+class LockManager:
+    def connect(self):
+        raise NotImplementedError
+
+    def acq4me(self, key, workers, maxqueue, timeout):
```

Comment:
Rename "lock" to keep the interface generic.

Shouldn't all arguments after "key" come from config?  Not sure how I feel about "key", but it can stay here for now.